content/en/docs/guides/wait-for-ready.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Wait For Ready
+description: >-
+  Explains how and why to configure the wait for ready feature.
+---
+
+### Overview
+
+This is a feature which can be used on a stub which will cause the rpcs to wait
+(until an optional deadline is exceeded) for the server to become available
+before sending the request.
+
+This allows for robust batch workflows as transient server problems (including
+startup) won't cause job failures.
+
+When an RPC is created while the channel is not in a READY state, without
+waitForReady it will immediately return a failure; with waitForReady it will
```

Comment:
It will depend on precisely what type of transient failure occurred.  This has been slightly rewritten.  What actually happens is that when an RPC comes along the channel tries to connect to the server and it is the failure of that which puts/keeps the subchannel and then the channel into/in TRANSIENT_FAILURE state.  The error that is returned has the details which are provided to the caller.